modules/dnn/test/imagenet_cls_test_alexnet.py

```diff
@@ -155,6 +155,20 @@ def get_output(self, input_blob):
         self.net.setInput(input_blob, self.in_blob_name)
         return self.net.forward(self.out_blob_name)
 
+class DNNOnnxModel(Framework):
+    net = object
+
+    def __init__(self, onnx_file, in_blob_name, out_blob_name):
+        self.net = cv.dnn.readNetFromONNX(onnx_file)
+        self.in_blob_name = in_blob_name
+        self.out_blob_name = out_blob_name
+
+    def get_name(self):
+        return 'DNN (ONNX)'
+
+    def get_output(self, input_blob):
+        self.net.setInput(input_blob, self.in_blob_name)
+        return self.net.forward(self.out_blob_name)
```

Comment:
This change is introduced from 4.x in https://github.com/opencv/opencv/pull/25435/files#diff-0f1c399b5f2a43b1c9501dd5fc4d97ae8306099eda327d3096cd84f5b22f2e9f